edvart/report_sections/table_of_contents.py

```diff
@@ -71,7 +71,7 @@ def _add_section_lines(
             for subsection in section.subsections:
                 self._add_section_lines(subsection, section_level + 1, lines, True)
 
-    # pylint: disable=arguments-differ
```

Comment:
Why is `differ` changed to `renamed`? `Section.add_cells()` has `df: pd.DataFrame` arg and this one has `cells: List[Dict[str, Any]]` arg.

Reply:
Because previously pylint was complaining about the different number of arguments. Now that `df` is added to the base method, the number of arguments is the same, just their names are different. I add `df` here because `TableOfContents` should not inherit from `Section` (see #79)